azure-pipelines.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ "x${devitoBackend}" == "xyask" ] || [ "x${devitoBackend}" == "xops" ] ; then
+  export DEVITO_BACKEND=${devitoBackend} ;
+  echo "Setting DEVITO_BACKEND=${DEVITO_BACKEND}" ;
+fi
+
+if [ "x${useOpenMP}" == 'xtrue' ] ; then
+  export DEVITO_OPENMP=1 ;
+  export OMP_NUM_THREADS=2 ;
+  echo "Setting DEVITO_OPENMP=${DEVITO_OPENMP} and OMP_NUM_THREADS=${OMP_NUM_THREADS}"
+fi
+
+if [ "x${testWithPip}" == 'xtrue' ] ; then
+  python setup.py test ; 
+else
+  source activate devito ;
+  py.test --cov devito tests/ ;
+  if [ ! -v DEVITO_BACKEND ] ; then 
+    python examples/seismic/benchmark.py test -P tti -so 4 -a -d 20 20 20 -n 5 ;
```

Comment:
all of my bash ignorance, again, so apologies in advance for the dumb question:

are you sure that **if** there's error in the first `python examples/...`, then it is actually reported? 

[here](https://github.com/opesci/devito/blob/master/.travis.yml#L119) we had to wrap each `python examples/...` within a separate ``if``

Reply:
The 'set -e' takes care of that, though it occurs to me on thinking about this again that this may not be the behaviour you want. The options are:

- Use 'set -e' so the script exits with error on the first error it encounters, or..
- Use 'set -o pipefail' so the script continues on error but exits with error if any command in the script returns an error

I guess you want the second of those behaviours ... ?